src/components/ha-sidebar.ts

```diff
@@ -441,6 +443,23 @@ class HaSidebar extends LitElement {
     fireEvent(this, "hass-toggle-menu");
   }
 
+  private _onPressed = (urlPath: string) => (_ev) => {
+    pressTimeout = setTimeout(() => {
+      this._handleExternalAppPanelLongPress(this.hass.panels[urlPath]);
```

Comment:
We shouldn't add such kind of interactions to the frontend. This means that the frontend now can never add any functionality here without having to consider the Android app. We don't want such kind of dependency.

Reply:
So is there a better way to do this or should I remove this functionality and perhaps add a settings screen to the android app?